docs/data-hub/dataframes.md

```diff
@@ -39,6 +39,23 @@ which should show you something like this:
     - Information about the last edit made to the database. 
 
 
+!!! tip "Fetching only some rows"
+    If you have a particularly large database, you may want to fetch only certain rows using a filter. For example, to fetch only rows where column `x` is greater than `42`:
+
+    ```python
+    from deeporigin.data_hub import filters
+    filter = filters.filter(
```

Comment:
Why do you need to do `filters.filter`?

Reply:
it's in its own module, which is weird. we'll fix when we have a nicer way of doing this